### PR TITLE
Add benchmarking example

### PR DESCRIPTION
**Pull request by clue**
*It is fast*. :shipit:

### CODE REVIEW

**Comment by clue** [comment]
My rather mediocre laptop (i3) can process around ~2000 requests/s or around 2000 MiB/s.

File: examples/99-benchmark-download.php [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+            $stream = new ChunkRepeater(str_repeat('.', 1000000), 1000);
+            break;
+        case '/10g.bin':
+            $stream = new ChunkRepeater(str_repeat('.', 1000000), 10000);
+            break;
+        default:
+            return new Response(404);
+    }
+
+    $loop->addTimer(0, array($stream, 'resume'));
```
Comment (WyriHaximus) [inline examples/99-benchmark-download.php]: Why not use a future tick instead here?

Comment (clue) [inline examples/99-benchmark-download.php]: Both would work here :+1: Using a timer here also makes this compatible with react/event-loop:^0.3 though.

**Comment by clue** [comment]
For the reference: The failing tests are unrelated and are being addressed in #163.

File: examples/99-benchmark-download.php (hunk 1 above)
Comment (WyriHaximus) [inline examples/99-benchmark-download.php]: Good point :+1:

**Comment by clue** [comment]
Rebased now that #163 is in :shipit:

**Pull request merged**